clarity_ext/context.py

```diff
@@ -263,12 +263,13 @@ def update(self, obj):
 
     def commit(self):
         """Commits all objects that have been added via the update method, using batch processing if possible"""
-        self._calls_to_commit += 1
-        if self._calls_to_commit > 1:
-            self.logger.warning("Commit called more than once. It's not necessary to call commit explicitly anymore.")
```

Comment:
This warning is perhaps still relevant in snpseq?

Reply:
It might be, but it has no effect to remove it either.